src/applications/financial-status-report/components/shared/ItemLoop.jsx

```diff
@@ -390,65 +388,69 @@ const ItemLoop = ({
       <div className="va-growable">
         <ScrollElement name={`topOfTable_${idSchema.$id}`} />
         {uiOptions.viewType === 'table' ? (
-          <table className="vads-u-font-family--sans vads-u-margin-top--3 vads-u-margin-bottom--0">
+          <va-table className="vads-u-margin-top--3 vads-u-margin-bottom--0">
```

Comment:
I've replaced `table` with `va-table` for this but I think that it will need to be refactored at some point because it includes a form within a table cell. This is not an appropriate [use case for table](https://design.va.gov/components/table#when-to-use-a-table). It's essentially just using the table for a layout.

Ideally the form would be outside of the table display but that work would be out of scope for this.